app/src/main/java/com/nativework/covid19vaccinetracker/ui/center/CenterActivity.kt

```diff
@@ -0,0 +1,99 @@
+package com.nativework.covid19vaccinetracker.ui.center
+
+import android.os.Bundle
+import androidx.recyclerview.widget.LinearLayoutManager
+import com.nativework.covid19vaccinetracker.R
+import com.nativework.covid19vaccinetracker.base.BaseApp
+import com.nativework.covid19vaccinetracker.databinding.ActivityCenterBinding
+import com.nativework.covid19vaccinetracker.models.Center
+import com.nativework.covid19vaccinetracker.utils.Constants
+
+class CenterActivity : BaseApp() {
+
+    private lateinit var binding: ActivityCenterBinding
+    private var adapter: CenterAdapter? = null
+
+    private var centerList = ArrayList<Center>()
+    private var freeList = ArrayList<Center>()
+    private var paidList = ArrayList<Center>()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        binding = ActivityCenterBinding.inflate(layoutInflater)
+        val view = binding.root
+        setContentView(view)
```

Comment:
Activity.actionBar.setUpHomeAsEnable(true)  or any other mechanism for backButton needed